hyperloglog.go

```diff
@@ -111,9 +124,10 @@ func (h *HyperLogLog) Count() uint64 {
 		if v > 0 {
 			estimate = m * math.Log(m/float64(v))
 		}
+	} else if estimate > 1.0/30.0*exp32 && withLargeRangeCorrection {
```

Comment:
nit: check the bool before the other condition to allow it to exit more quickly (not sure if this is a thing anymore)

Reply:
Depends which bool is most likely to be true. Either way, it should be negligible compared to the float operations that `count()` performs, so I think this is fine 😄